roxctl/deployment/check/check.go

```diff
@@ -107,14 +108,15 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	// Add all printer related flags
 	objectPrinterFactory.AddFlags(c)
 
+	c.Flags().StringSliceVar(&deploymentCheckCmd.files, "files", nil, "yaml file to send to Central to evaluate policies against")
 	c.Flags().StringVarP(&deploymentCheckCmd.file, "file", "f", "", "yaml file to send to Central to evaluate policies against")
```

Comment:
I don't understand why keeping both the singular `--file` and adding a new `--files` flag. Shouldn't single files be handled as a single element in `--files` parameter?

Reply:
See the post in the PR, I did not want to break existing scripts of customers using the `--file` flag if possible. If we combine the two perhaps it would make more sense under `--file`, but that also makes it not obvious that it can accept multiple files. Thoughts?

Reply:
What is the behavior of the slice? If the flag usage for a single element is identical to string argument then they are compatible and you can simply swap the flag. In this case, drop the `--files` and let `--file` accept multiple files.

Reply:
I know that `roxctl` is used often to automate tasks around ACS, so I'd be strongly against **removing** a parameter to not break existing automation of customers.

How about we take inspiration from `kubectl` and `helm`?
`Helm` is using the notation @janisz described: 
	`helm install ./path --values ./internalValues.yaml --values ./customSettings.yaml`
Whereas `kubectl` can apply a whole folder by providing `kubectl apply -f` with a directory instead of a file. It also supports `--recursive` to also apply all files in any subfolders of the given path.

Either way we wouldn't break existing functionality, and still be able to supply multiple files.
I'd be in favor of the Helm way. 

Here is some discussion involving how to do this in [Cobra](https://github.com/spf13/cobra/issues/661) and with the [flags package](https://stackoverflow.com/questions/28322997/how-to-get-a-list-of-values-into-a-flag-in-golang/28323276#28323276) that I dug up out of interest.